packages/ripple-tide-api/src/services/utils/mapping/body.js

```diff
@@ -32,7 +32,7 @@ export default {
                   ['field_paragraph_accordion_name'],
                   ''
                 ),
-                content: getBodyFromField(content, 'field_paragraph_accordion_body')
+                content: getBodyFromField(content, 'field_paragraph_accordion_body').html
```

Comment:
If we end up deprecating the way RplMarkup works with client side vue components, we can make getBodyFromField just return the html